packages/react-components/src/organisms/__tests__/TeamCreateOutputForm.test.tsx

```diff
@@ -154,19 +184,25 @@ describe('on submit', () => {
       </Router>,
     );
 
-    fireEvent.change(screen.getByLabelText(/url/i), {
-      target: { value: data.link },
-    });
-    fireEvent.change(screen.getByLabelText(/title/i), {
-      target: { value: data.title },
-    });
-    fireEvent.change(screen.getByLabelText(/description/i), {
-      target: { value: data.description },
-    });
-    userEvent.type(screen.getByLabelText(/Select the option/i), data.type);
-    fireEvent.keyDown(screen.getByLabelText(/Select the option/i), {
-      keyCode: ENTER_KEYCODE,
-    });
+    userEvent.type(screen.getByRole('textbox', { name: /url/i }), data.link!);
```

Comment:
I had this as fire event because its faster than typing AFAIK

Reply:
The code was:

```
userEvent.type(screen.getByLabelText(/Select the option/i), data.type);
    fireEvent.keyDown(screen.getByLabelText(/Select the option/i), {
      keyCode: ENTER_KEYCODE,
    });
```
I removed the fireEvent as you don't need it with out Dropdown.

```
userEvent.type(
      screen.getByRole('textbox', { name: /Select the option/i }),
      data.type,
    );

```